libs/application/core/src/types/Form.ts

```diff
@@ -5,7 +5,8 @@ import { MessageDescriptor } from 'react-intl'
 import { BoxProps } from '@island.is/island-ui/core'
 import { Application } from './Application'
 
-export type StaticText = (MessageDescriptor & { values?: object }) | string
+export type StaticTextObject = MessageDescriptor & { values?: object }
```

Comment:
Would be nicer to use `Record<string, unknown>` or probably somewhere a `RecordObject` exists somewhere iirc

Reply:
Good catch. I gave it a go and it resulted in some weird ts errors in other places that I'll tackle some other time.